src/wp-includes/media.php

```diff
@@ -2055,13 +2076,17 @@ function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
  * to remove the attribute or set it to another accepted value.
  *
  * @since 6.1.0
+ * @deprecated 6.4.0 Use wp_img_tag_add_loading_optimization_attrs() instead.
```

Comment:
It's important to improve the documentation for this function by adding a `@see` annotation, similar to how it's done for other `deprecated` functions.

```suggestion
 * @deprecated 6.4.0 Use wp_img_tag_add_loading_optimization_attrs() instead.
 * @see wp_img_tag_add_loading_optimization_attrs()
```